src/main/java/cs2103/v15_1j/jimjim/JJMain.java

```diff
@@ -31,12 +34,13 @@ public void start(Stage primaryStage) throws Exception {
 		Storage storage = new JJStorage();
 		Parser parser = new JJParser();
 		Searcher searcher = new JJSearcher();
+		Stack<Command> undoCommandHistory = new Stack<Command>();
 
 		storage.setSaveFiles(TASK_FILE_NAME, DEADLINE_TASK_FILE_NAME, EVENT_FILE_NAME);
 		con.setParser(parser);
 		con.setStorage(storage);
 		con.setSearcher(searcher);
-		
+		con.setUndoCommandHistory(undoCommandHistory);
```

Comment:
Why would you ever need this method? `undoCommandHistory` is always empty after initialization.